src/main/kotlin/dqualizer/dqexec/adapter/StimulusAdapter.kt

```diff
@@ -216,4 +231,12 @@ class StimulusAdapter(private val loadTestConstants: LoadTestConstants) {
         MEDIUM,
         HIGH
     }
+
+    private fun convertToTimeString(timeString: String): Duration? {
+        return Objects.requireNonNull(DurationParser.parseDuration(timeString))
+    }
```

Comment:
The return type does not quite match the semantic here, since the return value can never be null.
Kotlin's `requireNotNull` helps to keep it concise.

```suggestion
    private fun convertToTimeString(timeString: String): Duration {
        return requireNotNull(DurationParser.parseDuration(timeString))
    }
```